haaska.py

```diff
@@ -364,6 +402,15 @@ def set_percentage(self, val):
         brightness = (val / 100.0) * 255.0
         self._call_service('light/turn_on', {'brightness': brightness})
 
+    def set_color(self, hue, saturation, brightness):
+        red, green, blue = colorsys.hsv_to_rgb(hue, saturation, brightness)
```

Comment:
We aren't using YIQ, we are using HSV which doesn't seem to require the same scaling...